deployment/k8s-config/kustomize/base/skaha/config/add-user.yaml

```diff
@@ -0,0 +1,55 @@
+# Production version of add-user.
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{skaha.adduser.jobname}"
+spec:
+  ttlSecondsAfterFinished: 3600
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: "skaha-add-user"
+        image: images.canfar.net/skaha-system/add-user:1.2
+        imagePullPolicy: Always
```

Comment:
I think we should be some low resource requirements in here (and in the keel-dev one).  That will help ensure it's scheduled quickly.  You can add the clause:

```
        resources:
          requests:
            memory: "1Gi"
            cpu: "250m"
            ephemeral-storage: "1Gi"
          limits:
            memory: "4Gi"
            cpu: "1"
            ephemeral-storage: "2Gi"
```